src/c++/perf_analyzer/client_backend/client_backend.cc

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2020-2021, NVIDIA CORPORATION. All rights reserved.
+// Copyright (c) 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights
```

Comment:
Remove (c) to ensure it wraps in a single line.

Reply:
changed 👍